### PR TITLE
Update Export-Import-Platform.ps1

### PR DESCRIPTION
**Pull request by bab29**
Update to include -DisableSSLVerify

### CODE REVIEW

File: Platforms/Export-Import-Platform.ps1 [hunk 1]
```diff
@@ -74,6 +78,32 @@ Function Test-CommandExists
...
+	}
+}
+Else
+{
+	try{
+		Write-Debug "Setting script to use TLS 1.2"
+		[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12
+	} catch {
+		Write-Error "Could not change SSL settings to use TLS 1.2"
+		Write-Error (Join-ExceptionMessage $_.Exception) -ErrorAction "SilentlyContinue"
```
Comment (AssafMiron) [inline Platforms/Export-Import-Platform.ps1:104]: If I'm not mistaken there is no Join-ExceptionMessage in this script 
So need to replace with a different implementation or remove this line

File: Platforms/Export-Import-Platform.ps1 [hunk 2]
```diff
@@ -74,6 +78,32 @@ Function Test-CommandExists
...
+		Write-Warning "It is not Recommended to disable SSL verification" -WarningAction Inquire
+		# Using Proxy Default credentials if the Server needs Proxy credentials
+		[System.Net.WebRequest]::DefaultWebProxy.Credentials = [System.Net.CredentialCache]::DefaultCredentials
+		# Using TLS 1.2 as security protocol verification
+		[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12 -bor [System.Net.SecurityProtocolType]::Tls11
+		# Disable SSL Verification
+		[System.Net.ServicePointManager]::ServerCertificateValidationCallback = { $DisableSSLVerify }
+	} catch {
+		Write-Error "Could not change SSL validation"
+		Write-Error (Join-ExceptionMessage $_.Exception) -ErrorAction "SilentlyContinue"
```
Comment (AssafMiron) [inline Platforms/Export-Import-Platform.ps1:93]: Same comment here about the join-ExceptionMessage

File: Platforms/Export-Import-Platform.ps1 [hunk 3]
```diff
@@ -178,4 +208,4 @@ else
     Write-Error "This script requires PowerShell version 3 or above"
 }
 
-Write-Host "Export / Import Platform: Script Started" -ForegroundColor Cyan
\ No newline at end of file
+Write-Host "Export / Import Platform: Script Started" -ForegroundColor Cyan
```
Comment (AssafMiron) [inline Platforms/Export-Import-Platform.ps1:211]: Just noticed this, this is the last line of the script so needs to be changed from Started to Ended... :)

**Review by AssafMiron — CHANGES_REQUESTED** [review]

File: Platforms/Export-Import-Platform.ps1 (hunk 3 above)
Comment (bab29) [reply Platforms/Export-Import-Platform.ps1]: Fixed

File: Platforms/Export-Import-Platform.ps1 (hunk 2 above)
Comment (bab29) [reply Platforms/Export-Import-Platform.ps1:127]: Added in function Join-ExceptionMessage

File: Platforms/Export-Import-Platform.ps1 (hunk 1 above)
Comment (bab29) [reply Platforms/Export-Import-Platform.ps1:138]: Added in function Join-ExceptionMessage

**Review by AssafMiron — APPROVED** [review]

**Pull request merged**